### ISSUE TITLE
[Bug]: CLICKHOUSE_TRACKER_URL is unvaild

### ISSUE THREAD

**Issue by JonSmall** [civitai/civitai#544]
### What happened?

npm run dev

CLICKHOUSE_TRACKER_URL is unvaild

why use CLICKHOUSE_TRACKER_URL instead 

for (const key of Object.keys(_serverEnv.data)) {
error - (api)\src\env\server.mjs (13:8) @ eval
error - Error: Invalid environment variables
    at eval (webpack-internal:///(api)/./src/env/server.mjs:19:11)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
  11 | if (!_serverEnv.success) {
  12 |   console.error('❌ Invalid environment variables:\n', ...formatErrors(_serverEnv.error.format()));
> 13 |   throw new Error('Invalid environment variables');
     |        ^

### Steps to reproduce the problem

git check
git pull
npm install
npm run dev

### What should have happened?

show website

**Comment by JonSmall** [comment]
is there any stable release branch ? I use the main branch right now, lots of bugs , maybe someone merge branch goes wrong

**Comment by JustMaier** [comment]
Hi @JonSmall we haven't done a great job of keeping the database seeding script or the example env file up to date since we've been moving fast to add new features and the team usually operates with a shared set of environment variables and dev database. I apologize for the difficulties that presents when trying to pick up and run the codebase.

We're planning to migrate to a different ORM in the future and as part of that flatten the migration history. I believe that will solve the problem that you mentioned here. We'll also update the example env and ensure that things tied to internal operations of the site like ClickHouse remain optional.

**Comment by billcai** [comment]
Hi @JustMaier and team, thanks for the great work. Would it be possible for a schema dump (no data) in the mean time, with any sensitive data redacted? This would help to address several bug reports related to db issues.

**Comment by hassanRsiddiqi** [comment]
@billcai were you able to fix this issue? If yes can you please share? I am encountering the exact issue.

**Comment by hassanRsiddiqi** [comment]
yes, I had the issue, I resolve it after providing the click house details.
![Screen Shot 2023-07-06 at 12 11 00 AM](https://github.com/civitai/civitai/assets/23057904/4d1e6164-30f6-408c-b524-a2fdf2c0e722)

**Comment by hassanRsiddiqi** [comment]
Now I am facing this issue, I am stuck on it, and don't know how to fix. 
![Screen Shot 2023-07-06 at 12 11 38 AM](https://github.com/civitai/civitai/assets/23057904/7ca61ed7-ada1-4803-a860-b0896f67ff59)

**Comment by NicksonYap** [comment]
Solution is to use older commit:
https://github.com/civitai/civitai/commit/f5687d449f1be6200a2b209c38a04a0b52636815

Before these environment variables were introduced

In `.env` we can comment out `CLICKHOUSE_TRACKER_URL` and `SCHEDULER_ENDPOINT` to avoid that error, however it doesn't work for `ORCHESTRATOR_ENDPOINT` because:

this line is enforcing `ORCHESTRATOR_ENDPOINT` and `ORCHESTRATOR_ACCESS_TOKEN` to be required, although in `schema.mjs` it has been made optional

https://github.com/civitai/civitai/blob/bebed7ada44a1167d298403bbc01cbfa21941b4c/src/server/http/orchestrator/orchestrator.caller.ts#L13

```js
  static getInstance(): OrchestratorCaller {
    if (!env.ORCHESTRATOR_ENDPOINT) throw new Error('Missing ORCHESTRATOR_ENDPOINT env');
    if (!env.ORCHESTRATOR_ACCESS_TOKEN) throw new Error('Missing ORCHESTRATOR_ACCESS_TOKEN env');
//...
```

related: https://github.com/civitai/civitai/issues/566#issuecomment-1614760791
@manuelurenah

**Comment by zRich** [comment]
I am facing the same issue, any update?

**Comment by sovos15** [comment]
Should be closed if fixed by Environment fixes #1283
